service/service.go

```diff
@@ -602,14 +604,34 @@ func (s *Service) VerifyToken(ctx context.Context, req *pbsvc.UserRequest) (*pbs
 		return nil, status.Error(codes.InvalidArgument, consts.ErrNilRequestIdentification.Error())
```

Comment:
in the future avoid doing `identity := req.GetIdentification()` because you can mutate stuff. We are lucky that this is an external API called from a different memory space.